cypress/integration/images.spec.js

```diff
@@ -10,387 +10,242 @@ describe("Images", () => {
-    beforeEach(() => {
-      cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
-      window.localStorage.setItem("userId", "test")
-
-      cy.visit(`/sites/${TEST_REPO_NAME}/images`)
-    })
-
-    it("Images should contain albums and ungrouped images", () => {
-      cy.contains("Albums")
-      cy.contains("Ungrouped images")
-      cy.contains("Upload new image")
-      cy.contains("Create new album")
-    })
-
-    it("Should be able to create new image with valid title", () => {
-      // Set intercept to listen for POST request on server
-      cy.intercept({
-        method: "POST",
-        url: `/v1/sites/${TEST_REPO_NAME}/images`,
-      }).as("createImage")
-
-      cy.uploadMedia(IMAGE_TITLE, TEST_IMAGE_PATH)
-      // ASSERTS
-      cy.wait("@createImage") // should intercept POST request
-      cy.contains(IMAGE_TITLE, { timeout: CUSTOM_TIMEOUT }).should("exist") // image should be contained in Images
-    })
-
-    it("Should be able to edit an image", () => {
-      // Set intercept to listen for POST request on server
-      cy.intercept({
-        method: "POST",
-        url: `/v1/sites/${TEST_REPO_NAME}/images/${IMAGE_TITLE}/rename/${OTHER_IMAGE_TITLE}`,
-      }).as("renameImage")
-      cy.renameMedia(IMAGE_TITLE, OTHER_IMAGE_TITLE)
-      // ASSERTS
-      cy.wait("@renameImage")
-      cy.contains(OTHER_IMAGE_TITLE) // Image should be contained in Images
-
-      // Set intercept to listen for POST request on server
-      cy.intercept({
-        method: "POST",
-        url: `/v1/sites/${TEST_REPO_NAME}/images/${OTHER_IMAGE_TITLE}/rename/${IMAGE_TITLE}`,
-      }).as("renameImageBack")
-      cy.renameMedia(OTHER_IMAGE_TITLE, IMAGE_TITLE) // Rename image to original title
-      // ASSERTS
-      cy.wait("@renameImageBack")
-      cy.contains(IMAGE_TITLE, { timeout: CUSTOM_TIMEOUT }).should("exist")
-    })
-
-    it("Should not be able to create image with invalid title", () => {
-      // should not be able to save with invalid characters in title
-      cy.uploadMedia(INVALID_CHARACTER, TEST_IMAGE_PATH, ACTION_DISABLED)
-      // ASSERTS
-      cy.contains(
-        "Invalid filename: filename must not contain any special characters"
-      )
-      cy.get("button")
-        .contains(/^Upload$/)
-        .should("be.disabled") // necessary as multiple buttons containing Upload on page
-      cy.get("#closeMediaSettingsModal").click()
-
-      // title should not allow for names without extensions
-      cy.uploadMedia(MISSING_EXTENSION, TEST_IMAGE_PATH, ACTION_DISABLED)
-      // ASSERTS
-      cy.contains(
-        "Invalid filename: filename can only contain one full stop and must follow the structure {name}.{extension}"
-      )
-      cy.get("button")
-        .contains(/^Upload$/)
-        .should("be.disabled") // necessary as multiple buttons containing Upload on page
-      cy.get("#closeMediaSettingsModal").click()
+  // describe("Create image, delete image, edit image settings in Images", () => {
```

Comment:
@gweiying sorry to necro this but is this going to be used in the foreseeable future?